aeat.py

```diff
@@ -1155,6 +1166,8 @@ def get_received_sii_reports(cls):
                     ('sii_pending_sending', '=', True),
                     ('type', '=', 'in'),
                     ('move', '!=', None),
+                    ('invoice_date', '<=', today - timedelta(days=config.sii_default_limit_date)),
```

Comment:
Or we need to make "required" the config field, or ensure that it has value before set timedelta.
Use the new field name.